### DISCUSSION TITLE
When formula get updated

### DISCUSSION THREAD

**Discussion by alizufan** [Homebrew/discussions#4710] [Getting started] [answered]
### Output of `brew config`

```shell
not applicable
```

### Output of `brew doctor`

```shell
not applicable
```

### Description of issue

how long for formula get updated, i want update go to new version.
currenly go has 1.21 already, a lot new feature added and i want try it?

**Comment by osalbahr** [comment, marked as answer]
- https://github.com/Homebrew/homebrew-core/pull/134468

There is already an open pull request. You can either wait for it to be merged, or locally build the formula from source (see `brew help install`).

**Comment by alizufan** [comment]
Wonderfull, thanks for the response
